libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/api/MapboxRouteLineView.kt

```diff
@@ -76,7 +76,7 @@ import org.jetbrains.annotations.TestOnly
  * @param options resource options used rendering the route line on the map
  */
 @UiThread
-class MapboxRouteLineView(var options: MapboxRouteLineOptions) {
+class MapboxRouteLineView(val options: MapboxRouteLineOptions) {
```

Comment:
This is a breaking change but I understand that we didn't intend to expose the setter in the first place. Problem is that it could actually be used in some circumstances and work, unlike some of the other data class that we've hidden from public API in the past as they server no practical purpose.

What if we did this instead:
```suggestion
class MapboxRouteLineView(options: MapboxRouteLineOptions) {

    var options: MapboxRouteLineOptions = options
        @Deprecated(
            message = "Avoid using this setter as " +
                "it will not correctly re-apply all mutated parameters."
        )
        set
```

Reply:
This line currently matches what's in Main.

Reply:
`main` defines a `var`:
https://github.com/mapbox/mapbox-navigation-android/blob/c8bc7ab2fa9037a970615fdc55ac5a70bda93622/libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/api/MapboxRouteLineView.kt#L79

Reply:
@cafesilencio the comment above refers to `MapboxRouteLineView` but the currently proposed changes are done to `MapboxRouteArrowView`. We should not be changing `MapboxRouteArrowView`.

Reply:
My mistake, I don't know what I was looking at.

Reply:
I adjusted the code.